### PR TITLE
route pages

### PR DESCRIPTION
**Pull request by YuliUA**
@wizzardio Check pls...

### CODE REVIEW

File: router/src/components/api.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	addNewPlayer: function(number, name, position) {
+		this.players.push({
+			number: number,
+			name: name,
+			position: position
+		});
+	},
+	deleteHandle: function(e) {
+		e.preventDefault();
+		let name = e.target.parentElement.firstChild.innerText;
```
Comment (wizzardio) [inline router/src/components/api.js]: why do yo use e.target.parentElement.firstChild and preventDefault?

File: router/src/components/api.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		let index = this.players.findIndex(i=>i.name===name)
```
Comment (wizzardio) [inline router/src/components/api.js]: need to use ID since name is not equal

File: router/src/components/FullPlayers.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        }
+        this.delete = this.delete.bind(this);
+        this.change=this.change.bind(this);
+        this.addPlayerForm=this.addPlayerForm.bind(this);
+    }
+
+    delete(e) {
+        let newPlayers = this.state.players.filter(p => p.number.toString() !== e.target.id);
+        // console.log(newPlayers)
+        console.log(PlayerAPI.deleteHandle(e))
```
Comment (wizzardio) [inline router/src/components/FullPlayers.js]: why?

File: router/src/components/FullPlayers.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        this.setState({
+            players: newPlayers
+        })
+    }
+    handleSubmit(){
+        let name=this.state.playerName
+        let number=parseInt(this.state.playerNumber,10)
```
Comment (wizzardio) [inline router/src/components/FullPlayers.js]: if you want to have ID for each player, try this -> 
npm install uuid
var uuid = require('uuid');

just like we did in firebase example

Comment (YuliUA) [reply router/src/components/FullPlayers.js]: @wizzardio I've made some changes (add id to each player & rewrite deleteHandle function) check it)

**Pull request merged**